vep-mapping-pipeline/repeat_expansion_variants/pipeline.py

```diff
@@ -208,7 +240,23 @@ def main(clinvar_xml, output_consequences, output_dataframe):
             all relevant columns and can be used for review or debugging purposes."""
 
     logger.info('Load and preprocess variant data')
-    variants = load_clinvar_data(clinvar_xml)
+    variants, s = load_clinvar_data(clinvar_xml)
+
+    # Output ClinVar record statistics
+    logger.info(f'''
+        Microsatellite records: {sum(s.values())}
+            With complete coordinates: {s[clinvar_xml_utils.ClinVarRecordMeasure.MS_DELETION] +
+                                        s[clinvar_xml_utils.ClinVarRecordMeasure.MS_SHORT_EXPANSION] +
+                                        s[clinvar_xml_utils.ClinVarRecordMeasure.MS_REPEAT_EXPANSION]}
+                Deletions: {s[clinvar_xml_utils.ClinVarRecordMeasure.MS_DELETION]}
+                Short insertions: {s[clinvar_xml_utils.ClinVarRecordMeasure.MS_SHORT_EXPANSION]}
+                Repeat expansions: {s[clinvar_xml_utils.ClinVarRecordMeasure.MS_REPEAT_EXPANSION]}
+            No complete coordinates: {s[clinvar_xml_utils.ClinVarRecordMeasure.MS_NO_COMPLETE_COORDS]}
+    '''.replace('\n' + ' ' * 8, '\n'))
```

Comment:
if I understand correctly, these stats are preliminary. You don't have yet the number of `trinucleotide_repeat_expansion` and `short_tandem_repeat_expansion`. 
Wouldn't it make sense to have these number in the logs?

Reply:
That's right, good point. I've now added output of those stats as well: https://github.com/EBIvariation/eva-opentargets/pull/194/commits/1ba3df54858546a7b62a9728b27b38ff91d624ab